test-data/unit/cmdline.pyproject.test

```diff
@@ -0,0 +1,82 @@
+-- Tests for command line parsing
+-- ------------------------------
+--
+-- The initial line specifies the command line, in the format
+--
+--   # cmd: mypy <options>
+--
+-- Note that # flags: --some-flag IS NOT SUPPORTED.
+-- Use # cmd: mypy --some-flag ...
+--
+-- '== Return code: <value>' is added to the output when the process return code
+-- is "nonobvious" -- that is, when it is something other than 0 if there are no
+-- messages and 1 if there are.
+
+-- Directories/packages on the command line
+-- ----------------------------------------
```

Comment:
Could you add back some successful test cases as well that would cover all the essential features? It looks like all the test cases test failure modes.

Please add additional test cases to `check-*.test` files, since they are much faster than command-line tests.

Reply:
Sorry, got a little overzealous in the removal. Will do.